docs/faq.md

```diff
@@ -32,19 +32,73 @@ See the Semgrep feature and pricing comparison at [r2c.dev/pricing](https://r2c.
 
-Semgrep incorporates rulesets inspired by the rules of many popular linters and checkers, including ESLint, RuboCop, Bandit, and FindSecBugs. But Semgrep also allows you to enable multiple rulesets at the same time without adding linter-specific artifacts or installation to your code repository.
+To our knowledge, the only other tool with the explicit goal of allowing custom rules is Semmle’s proprietary tool, CodeQL. CodeQL has a domain-specific language which is extremely powerful but is designed for those with significant program analysis expertise, whereas Semgrep is designed for the security engineer* ** *or developer who wants to automate code review. Our goal is to make writing a Semgrep rule as easy as copying the code you want to find—and letting the Semgrep engine to make the rule and autofix high-quality enough to run in CI or the editor.
 
-Lastly, while many linters are extensible, with Semgrep you don’t need to learn specific abstract syntax tree (AST) based patterns for writing custom rules. You can read more about the spectrum from linters to complex analysis tools in Instagram's wonderful article [Static Analysis at Scale: An Instagram Story](https://instagram-engineering.com/static-analysis-at-scale-an-instagram-story-8f498ab71a0c).
+Our hosted offering, [Semgrep.dev](http://semgrep.dev/), has a generous free tier (even for private repos!) that offers a hosted CI integration with one-click setup and diff-awareness so you can start running Semgrep right away on new code so that you don’t have to fix all the existing issues when you first start out. For users running inside orgs with lots of repos, the hosted offering also offers a policy and notification system that makes it easy to tune Semgrep so that it only reports issues or suggest fixes that actually get applied. Our goal is a 99% fix rate for what Semgrep reports.
+
+### Besides open-source and ease of writing new rules, what else is different about Semgrep?
+
+**1. Speedy & offline: Semgrep runs offline on every keystroke**
+
+If you are shipping code daily a code analysis tool that takes a week is not helpful. We think modern static analysis tools should run on every keystroke in the editor, without needing network access. Semgrep runs at approximately 20K-100K loc/sec per rule but our goal is to be even faster.
+
+
+**2. Semantic: Semgrep is smart**
+
+Semgrep automatically handles the nuance of “there’s more than one way to do it”: you write your query and all equivalent variations of that code should be automatically matched. 
+
+As Semgrep evolves, a query like *foo(“password”*) becomes smarter. In the original version of Semgrep, this query would only match the code `foo("password")`. But a few months after release it would match `const x = "password"; foo(x).` Today Semgrep can [do even more with intraproceedural dataflow](https://semgrep.dev/s/ievans:c-dataflow) analysis, and we’re working on adding more of these semantic features with every release.
```

Comment:
```suggestion
As Semgrep evolves, a query like *foo(“password”)* becomes smarter. In the original version of Semgrep, this query would only match the code `foo("password")`. But a few months after release it would match `const x = "password"; foo(x).` Today Semgrep can [do even more with intraproceedural dataflow](https://semgrep.dev/s/ievans:c-dataflow) analysis, and we’re working on adding more of these semantic features with every release.
```